content/en/docs/instrumentation/python/getting-started.md

```diff
@@ -1,285 +1,268 @@
 ---
 title: Getting Started
 weight: 1
-code_block_from__path_base: content-modules/opentelemetry-python/docs/getting_started
 ---
 
-This guide walks you through instrumenting a Python application with
-`opentelemetry-python`.
+In this page, you'll learn how to set up and get tracing telemetry from an HTTP
+server with Flask. If you're not using Flask, that's fine - this guide will also
+work with Django, FastAPI, [and
+more](https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation).
 
 For more elaborate examples, see
 [examples](https://github.com/open-telemetry/opentelemetry-python/tree/main/docs/examples/).
 
-## Hello world: emit a trace to your console
+## Installation
 
-To get started, install both the opentelemetry API and SDK:
+To begin, set up an environment and install dependencies:
 
-```sh
-pip install opentelemetry-api
-pip install opentelemetry-sdk
+```console
+$ mkdir otel-getting-started
+$ cd otel-getting-started
+$ python3 -m venv .
+$ source ./bin/activate
 ```
 
-The API package provides the interfaces required by the application owner, as
-well as some helper logic to load implementations.
+Now install the HTTP server:
```

Comment:
is "HTTP server" really the correct description for Flask?

Reply:
I'll clean this up, I think at some point in the future we'd want to also cover Django, at which point wording would have to change up a bit. But that's a problem for another day